src/sentry/utils/performance_issues/detectors/experiments/n_plus_one_db_span_detector.py

```diff
@@ -237,28 +229,23 @@ def _is_slower_than_threshold(self) -> bool:
         duration_threshold = self.settings.get("duration_threshold")
         return total_span_time(self.n_spans) >= duration_threshold
 
-    def _contains_valid_repeating_query(self, span: Span) -> bool:
-        # Make sure we at least have a space, to exclude e.g. MongoDB and
-        # Prisma's `rawQuery`.
-        query = span.get("description", None)
-        return bool(query) and " " in query
-
```

Comment:
So I took apart this function and moved it's check to the `visit_span` method via `are_spans_equivalent`.